### PR TITLE
fix comment

### PR DESCRIPTION
**Pull request by iuhilnehc-ynos**
Just a tiny fix

### CODE REVIEW

**Review by fujitatomoya — APPROVED** [review]

File: rcl/include/rcl/subscription.h [hunk 1]
```diff
@@ -104,7 +104,7 @@ rcl_get_zero_initialized_subscription(void);
  * ```
  *
  * The rosidl_message_type_support_t object contains message type specific
- * information used to publish messages.
+ * information used to subscribe messages.
```
Comment (clalancette) [inline rcl/include/rcl/subscription.h]:
```suggestion
 * information used to subscribe to messages.
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rcl/include/rcl/subscription.h (hunk 1 above)
Comment (iuhilnehc-ynos) [reply rcl/include/rcl/subscription.h]: Thank you. Updated in https://github.com/ros2/rcl/pull/1073/commits/8d499463e64fb07507d431e0c7112934d5d33082

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Comment by iuhilnehc-ynos** [comment]
CI:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/18844/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/13384/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/19596/)

**Pull request merged**